.github/workflows/frontend-build-docker.yml

```diff
@@ -1,69 +0,0 @@
-name: Create and publish Docker images from the frontend directory
```

Comment:
This file appears to have been a merge conflict.. and I have deleted this duplicate.